src/styles/variables.js

```diff
@@ -75,4 +75,5 @@ export const menuZIndex = '170'; // menu-z-index
 export const notificationsWrapperZIndex = '180'; // notifications-wrapper-z-index
 export const overlayZIndex = '190'; // overlay-z-index
 export const tooltipZIndex = '190'; // tooltip-z-index
+export const defaultLineHeight = '20px'; // default-line-height
```

Comment:
hmm, I would have expected the name to be `bdlLineHeight`. Did you add these two (the JS variable, and the corresponding JSON) manually?

Reply:
This was automatically generated

Reply:
On that point thought, I'm open to calling it either "default" or "global".

Reply:
The name here and in the JSON should match a variable in _variables.scss, but I don't see one in here with the `default-line-height` name 🤔 (this happens in build-style-vars.js)

Can you re-run `build:variables` to double-check?

Reply:
Sure I ran it again and got a diff on pretty much the whole file. I'll push that up so you can check it out.

Reply:
Oddly enough, git locally gives me a much different diff than when I push to GHE. Might just be git being weird. Seems like it's the same as before.